git-flow-init

```diff
@@ -374,6 +374,17 @@ file=    use given config file
 		git_do config $gitflow_config_option gitflow.path.hooks "$hooks_dir"
 	fi
 
+    # Automate Pre-Commit hook install if it exists on git-flow hooks directory
+    hooks_dir=$(git config --get gitflow.path.hooks)
+    if [ -f "$hooks_dir/pre-commit" ]; then
+        # Check whether there's a previous local pre-commit hook 
+        if [ -f "./.git/hooks/pre-commit" ]; then
+            echo "A backup of your local pre-commit hook will be created."
+        fi
+        ln -s "$hooks_dir/pre-commit" -t "./.git/hooks" --backup
```

Comment:
Hi Peter,
When git-flow hooks files are placed outside the repo the `$DOT_GIT_DIR` variable won't work.

For `$ git config gitflow.path.hooks /path/to/git-flow-hooks`, `$DOT_GIT_DIR` will be `/path/to/git-flow-hooks` what breaks my test.

Was it clear?
Am I missing something?

Thanks for you feedback